### PR TITLE
Changed SR650 port names

### PR DESCRIPTION
**Pull request by BenMatase**

### CODE REVIEW

**Review by haberkornsam — APPROVED** [review]

File: generate_templates.py [hunk 1]
```diff
@@ -207,24 +208,35 @@ def main():
...
                 continue
-            template = generate_template(vendor, sku, actual_devicemap)
-            write_template(vendor, sku, template)
 
-def generate_template(vendor, sku, devicemap):
+            model = devicemap.get('displayModel') or sku
+
+            # make sure models are unique per vendor
+            if model in models:
+                print("Vendor {vendor} has duplicate model {model}")
```
Comment (haberkornsam) [inline generate_templates.py:221]: should be an fstring

File: generate_templates.py [hunk 2]
```diff
@@ -207,24 +208,35 @@ def main():
...
+                sys.exit(1)
```
Comment (haberkornsam) [inline generate_templates.py:222]: Should we actually exit here? I think we should just not generate a template for the duplicate. I could see the `displayModel` being used for something like the madrids where there are a whole bunch of them just slightly different.

File: generate_templates.py [hunk 3]
```diff
@@ -207,24 +208,35 @@ def main():
...
+        models = set()
         for sku, devicemap in skus.items():
             actual_devicemap = resolve_alias(devicemap, interfacemap)
             if not actual_devicemap:
                 continue
-            template = generate_template(vendor, sku, actual_devicemap)
-            write_template(vendor, sku, template)
 
-def generate_template(vendor, sku, devicemap):
+            model = devicemap.get('displayModel') or sku
```
Comment (haberkornsam) [inline generate_templates.py:217]: nit: `.get("displayModel", sku)`

File: generate_templates.py [hunk 4]
```diff
@@ -207,24 +208,35 @@ def main():
...
+
+            template = generate_template(vendor, model, actual_devicemap)
+            write_template(vendor, model, template)
+
+def generate_template(vendor, model, devicemap):
     return {
-        "name": f"{vendor}-{sku}-Template",
-        "description": f"Adds a standalone {vendor} {sku} router: {devicemap.get('description', '')}",
+        "name": f"{vendor}-{model}-Template",
+        "description": f"Adds a standalone {vendor} {model} router: {devicemap.get('description', '')}",
```
Comment (haberkornsam) [inline generate_templates.py:232]: I know not a part of your change,  but can you conditionalize the `:` in the description? If there is not a description from the devicemap, it leaves a hanging colon.

File: devicemaps.json [hunk 5]
```diff
@@ -1443,81 +1443,82 @@
     },
     "Lenovo": {
       "7Z73CTO1WW": {
+        "displayModel": "SR650",
```
Comment (haberkornsam) [inline devicemaps.json:1446]: Can we also add a description for the device?

**Review by haberkornsam — APPROVED** [review]
Overall looks good

File: generate_templates.py (hunk 2 above)
Comment (BenMatase) [reply generate_templates.py:222]: I disagree I think. We don't want certain devices to not have a unique template. Or else, someone could use a template for another device that isn't appropriate

Comment (BenMatase) [reply generate_templates.py:222]: And discussed offline, we can change this behavior in the future if we run into a roadblock

**Pull request merged**